lib/routes/profile.js

```diff
@@ -60,7 +67,21 @@ module.exports = {
       } else {
         logger.info('batch.db');
       }
+
       return rep.header('last-modified', lastModified.toUTCString());
+    }
+
+    server.methods.profileCache.get(req, (err, result, cached, report) => {
+      // Check to see if the auth-server is reporting a newer `profileChangedAt`
+      // than the cached value. If so, lets invalidate the cache and set new value.
+      if (result && result.profileChangedAt < creds.profileChangedAt) {
```

Comment:
It seems a bit weird to test for the existence of `result` here, since IIUC it will only be missing when there has been an error.  Would either an early return, or an `if (!err)` be clearer here?